### ISSUE TITLE
What level is pretrained large20 against random? The win rate is always 0.5

### ISSUE THREAD

**Issue by arisliang** [yhyu13/AlphaGOZero-python-tensorflow#16]
The win rate is always 0.5 between large 20 and random, even approaching the end. Is the pretrained model no particular difference than random, even though it's running DNN+MCTS?

![screenshot from 2018-04-30 09-09-38](https://user-images.githubusercontent.com/1115503/39412978-8476ed08-4c56-11e8-9d9f-3a5cc284019a.png)

**Comment by yhyu13** [comment]
@arisliang 

I tested it and found the same problem. The answer from me is that the MCTS player is broken in some part. If you want to test the greedy policy (DNN top1),  you can pull the lasted commit.

**Comment by arisliang** [comment]
Pulled the updates, greedy policy works. But it doesn't give win rate. Plus it doesn't seem to me the greedy policy has advantage against random.

![screenshot from 2018-05-01 10-09-41](https://user-images.githubusercontent.com/1115503/39458533-f6bb7a6a-4d27-11e8-925e-4b3254418d86.png)
